src/payment/checkout/Checkout.jsx

```diff
@@ -74,14 +74,13 @@ class Checkout extends React.Component {
       intl,
       isFreeBasket,
       loading,
-      isBasketProcessing,
       isPaymentVisualExperiment,
       paymentMethod,
       submitting,
       orderType,
     } = this.props;
 
-    const submissionDisabled = loading || isBasketProcessing;
+    const submissionDisabled = submitting;
```

Comment:
I think achieving the editability of form fields this way technically would allow form submission during load. This is unlikely to be an issue,  but an alternative may be to create a new `editable` prop instead of using `submissionDisabled` as the way to disable form fields.

Reply:
Is form submission possible even when the submit button is disabled?

Reply:
Only through someone editing the HTML or something like that. It's always going to be possible to submit the form client side, but the backend handles that so I don't think it's a big concern

Reply:
I will disable the submit button on bulk purchases to prevent the above issue @bbaker6225, thank you!